staging/kos/pkg/networkfabric/interfaces.go

```diff
@@ -18,33 +18,116 @@ package networkfabric
 
 import "net"
 
-// Interface is the interface of a network fabric which allows
-// the user to implement Netowrk Interfaces. The implementer
-// MUST return nil when the user attempts to delete an Interface
-// which does not exist.
+// Interface is the contract of a VXLAN network fabric.
```

Comment:
"Interface" is a very reasonable name in general, but in this context is confusing because we also have the prior term "network interface".

Reply:
How about "Contract"?

Reply:
switched to "Contract" in bfbf60c

Reply:
What about "Manager" or "Factory" or "InterfaceFactory"?

Reply:
"Manager" seems good enough to me. "Factory"  would be confusing, as we also have a NetworkFabric factory.

Reply:
Actually, I think "Contract" is better than "Manager". The package name is "networkfabric". For importers explicitly using the network fabric interface (as in, programming language interface), "networkfabric.Contract" is less misleading than "networkfabric.Manager". I think the latter might be interpreted as "a manager of networkfabrics", which is not, it's a manager of network interfaces.

We could rename the package to "networkinterface" and the networkfabric interface to "Manager". This way, importers would use "networkinterface.Manager", that conveys the right idea.

Reply:
I like the package name networkfabric.  There is more to the fabric than the network interfaces.

What about `networkfabric.InterfaceManager`?

Reply:
Ok.

Reply:
92694a5 does the update.